android/app/src/main/java/updated/mysterium/vpn/ui/create/account/CreateAccountActivity.kt

```diff
@@ -87,8 +87,11 @@ class CreateAccountActivity : BaseActivity() {
     private fun uploadKey() {
         val intent = Intent(Intent.ACTION_GET_CONTENT)
         intent.type = "*/*"
-        val mimeTypes = arrayOf(MIME_TYPE_JSON)
-        intent.putExtra(Intent.EXTRA_MIME_TYPES, mimeTypes)
+        if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.P) {
+            intent.putExtra(Intent.EXTRA_MIME_TYPES, arrayOf("application/octet-stream"))
+        } else {
+            intent.putExtra(Intent.EXTRA_MIME_TYPES, arrayOf("application/json"))
+        }
```

Comment:
Let's make it by constants. Something like this:
private companion object {
         const val MIME_TYPE_JSON = "application/json"
         const val MIME_TYPE_BINARY_FILE = "application/octet-stream"
}

Reply:
fixed in 42a3a06a